src/main/java/seedu/event/Event.java

```diff
@@ -77,15 +77,15 @@ public void setName(String name) throws InvalidParameterException {
      * @return the datetime of the event
      */
     public String getDatetime() {
-        return this.datetime;
+        return datetime.getDateTimeFormat();
```

Comment:
Will using this.datetime.getDateTimeFormat() be better? Make it more explicit that the datetime variable comes from the class.